MoneyDataType/ICurrency.cs

```diff
@@ -7,27 +7,36 @@ namespace OrchardCore.Commerce.Abstractions
     /// </summary>
     public interface ICurrency : IEquatable<ICurrency>
     {
+        /// <summary>
+        /// Culture name for the currency
+        /// </summary>
+        string Culture { get; }
+
         /// <summary>
         /// The symbol for the currency, usually a single character
         /// </summary>
         string Symbol { get; }
+
         /// <summary>
-        /// The full name of the currency
+        /// The full english name of the currency
         /// </summary>
-        string Name { get; }
+        string EnglishName { get; }
```

Comment:
This comes from RegionInfo, and since we created our own version I thought it relevant to persist this value.  Also the Native and English both have there needs when talking about currencies.

Reply:
There's a lot more we don't persist. As mentioned in another comment, we should either make it fully localizable or remove. Let's remove for now.

Reply:
fixed